app/MapScreen.js

```diff
@@ -65,16 +199,122 @@ export default class MapScreen extends Component {
     }
     tileQueryParams['eo:bands'] = bandCombinations[bandType];
 
+    const tileQueryParamsUI = {
+      bandCombination: bandType,
+      date: endDate,
+      cloudPercentage: lexSlotValues.CloudPercentage || 0,
+    };
+
     this.setState({
+      tileQueryParamsUI,
       tileQueryString: queryString.stringify(tileQueryParams),
     });
   }
 
+  finishRecording(filePath) {
+    this.setState({
+      isRecording: false,
+    });
+    console.log(`Finished recording at path: ${filePath}`);
+  }
+
+  prepareRecorder() {
+    const audioPath = `${AudioUtils.DocumentDirectoryPath}/test2.lpcm`;
```

Comment:
One thing I ran into at some point with testing was that if I recorded over the `test2.lpcm` file too many times, it stopped working. So I thought about using the date/timestamp as the filename or something. This may be an unnecessary optimization for now though

Reply:
Interesting, let's make an issue for that and watch for it.

Reply:
OK, opened #8 for this